charts/restate-helm/templates/statefulset.yaml

```diff
@@ -134,7 +134,8 @@ spec:
     - metadata:
         name: storage
         labels:
-          {{- include "restate.labels" . | nindent 10 }}
```

Comment:
Does this problem apply to other created resources as well (services, volume claims, etc.)?

Reply:
No, statefulset has famously annoying immutability. Everything else the labels will just update.